source/administration/config-settings.rst

```diff
@@ -2437,6 +2445,15 @@ See the `Guest Accounts documentation <https://docs.mattermost.com/deployment/gu
 | This feature's ``config.json`` setting is ``"GuestAttribute": ""`` with string input.                                                                                |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Admin Attribute
+^^^^^^^^^^^^^^^^^
+*Available in Enterprise Edition E20*
+
+(Optional) The attribute in the SAML Assertion for designating System Admins. The user is promoted to this role on their next login.
+
```

Comment:
Same question as the AD/LDAP filter - what happens if an admin does not match the attribute?

Reply:
I think the suggested wording for above works here too:
"If Admin access is set to false, users who are currently logged in retain their Admin role. When they log out this is revoked and on their next login they will no longer have Admin privileges."

Reply:
Adding @thefactremains and @sbishel to this for their input. :)

Reply:
Same here - 
"EnableAdminFilter": false,
"AdminFilter": "",